src/icon-tab-bar.scss

```diff
@@ -751,6 +790,29 @@ $fd-icon-tab-bar-semantic-values: (
 
         font-size: 1rem;
       }
+
+      &--positive {
+        @include fd-icon-tab-bar-list-item-color(var(--sapPositiveElementColor));
+      }
+
+      &--negative {
+        @include fd-icon-tab-bar-list-item-color(var(--sapNegativeElementColor));
+      }
+
+      &--critical {
+        @include fd-icon-tab-bar-list-item-color(var(--sapCriticalElementColor));
+      }
+
+      &--informative {
+        @include fd-icon-tab-bar-list-item-color(var(--sapInformativeElementColor));
+      }
```

Comment:
should be documented as breaking changes as they are new classes.

Reply:
Can't you add the colors to `$fd-icon-tab-bar-semantic-values` mixin at the top of the file and then use a loop to set the color?

Reply:
Breaking changes were added to description of PR and Breaking changes page.